src/VisualStudio/Core/Def/ColorSchemes/VisualStudio2017.xml

```diff
@@ -63,6 +63,10 @@
             </Color>
             <Color Name="label name">
             </Color>
+            <Color Name="inline parameter name hints">
```

Comment:
@JoeRobich We still need this in both the 2017 and 2019 files, even if we're not also changing this between our classic/enhanced? I wasn't sure if the intent was these files are only for the classification schemes or something more general.

Reply:
Each file is its own self contained world, so they need to be defined in both places. If we wanted to say these colors are not going to be scheme-able we could move this to the VS EditorColors.xml. But really, I should finish the job of moving every Roslyn color into these schemes.